docker/docker-entrypoint.sh

```diff
@@ -137,7 +137,7 @@ initialize() {
 install_languages() {
 	echo "Installing languages..."
 
-	local langs="$1"
+	local -r langs="$1"
```

Comment:
IS it really necessary to:
- Declare a local readonly variable
- To only be used once

Wouldn't
read -ra langs <<<"$1" be enough?

Of course it might be a good idea to introduce some input checks

Reply:
Okay, I might be picky here, so I'll resolve